superset-frontend/src/dashboard/components/AddSliceCard/AddSliceCard.tsx

```diff
@@ -262,10 +263,16 @@ const AddSliceCard: React.FC<{
               `}
             >
               <MetadataItem label={t('Viz type')} value={vizName} />
-              <MetadataItem
-                label={t('Dataset')}
-                value={<a href={datasourceUrl}>{datasourceName}</a>}
-              />
+              {datasourceUrl && (
```

Comment:
Can you display the `datasourceName` when `datasourceUrl` is `undefined`?

Reply:
Done